src/lib.rs

```diff
@@ -194,10 +194,12 @@ mod tests {
                     policy_path
                         .with_extension("cil")
                         .file_name()
-                        .expect(&format!(
-                            "failed to extract file name from `{}`",
-                            policy_path.to_string_lossy()
-                        )),
+                        .unwrap_or_else(|| {
```

Comment:
I don't get this suggestion, which version of Clippy do you use?

Reply:
clippy 0.1.56 (59eed8a 2021-11-01)